core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1059,16 +1046,16 @@ class DAGScheduler(
 
         if (disallowStageRetryForTest) {
           abortStage(failedStage, "Fetch failure will not retry stage due to testing config")
-        } else if (failedStages.isEmpty && eventProcessActor != null) {
+        } else if (failedStages.isEmpty && eventProcessLoop != null) {
```

Comment:
I am not certain the null check is necessary now that this is a `val` declared in the constructor.